org.eclipse.lemminx/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -50,6 +50,12 @@
 		{
 			"name": "org.apache.xerces.impl.msg.XMLMessages"
 		},
+        {
+            "name": "com.thaiopensource.datatype.xsd.resources.Messages"
+        },
+        {
+            "name": "com.thaiopensource.relaxng.pattern.resources.Messages"
+        },		
```

Comment:
While using *.rnc files with the binary I get an Exception with "Can't find bundle for base name com.thaiopensource.relaxng.parse.compact.resources.Messages". I think adding this to this file should fix it:

```json
{
   "name": "com.thaiopensource.relaxng.parse.compact.resources.Messages"
},
```

Reply:
Thanks, fixed.